include/deal.II/fe/fe_interface_values.h

```diff
@@ -162,6 +168,20 @@ class FEInterfaceValues
   bool
   at_boundary() const;
 
+  /**
+   * Mapped quadrature weight. This value equals the
+   * mapped surface element times the weight of the quadrature
+   * point.
+   *
+   * You can think of the quantity returned by this function as the
+   * surface element $dds$ in the integral that we implement here by
```

Comment:
Why do you write `$dds$`? Did you mean `$ds$` as [here](https://github.com/dealii/dealii/blob/master/include/deal.II/fe/fe_values.h#L2943)?

Reply:
Copy-paste mistake...